prepare/augmenters/text_augmentors.py

```diff
@@ -0,0 +1,31 @@
+from unitxt.augmentors import (
+    AugmentPrefixSuffix,
+    AugmentWhitespace,
+)
+from unitxt.catalog import add_to_catalog
+
+pattern_distribution = {
+    " ": 20,
+    "\\t": 10,
+    "\\n": 40,
+    "": 30,
+}
+
+operator = AugmentPrefixSuffix(
+    prefixes=pattern_distribution,
+    suffixes=pattern_distribution,
+    prefix_len=5,
+    suffix_len=5,
+    remove_existing_whitespaces=True,
+)
+
+add_to_catalog(operator, "augmentors.text.prefix_suffix", overwrite=True)
```

Comment:
They should be ""augmentors.text.whitespace_prefix_suffix" (There could be other prefix_suffix augmenters, that add other things.